include/oboe/FullDuplexStream.h

```diff
@@ -24,26 +24,68 @@
 
 namespace oboe {
 
-class FullDuplexStream : public AudioStreamCallback {
+/**
+ * FullDuplexStream can be used to synchronize an input and output stream.
+ *
+ * For the builder of the output stream, call setDataCallback() with this stream.
```

Comment:
Hmm. Is this a stream or a callback? I guess it's both.
Maybe say "with this object"?